tavern/testutils/helpers.py

```diff
@@ -90,17 +90,12 @@ def validate_regex(response, expression):
         response (Response): reqeusts.Response object
         expression (str): Regex expression to use
     Returns:
-        dict: dictionary of regex: boxed full_match, and list of matched groups
+        dict: dictionary of regex: boxed name capture groups
     """
     match = re.search(expression, response.text)
 
     assert match
 
-    full_match = match.group(0)
-    groups = [group for group in match.groups()]
     return {
-        "regex": Box({
-            'full_match': full_match,
-            'groups': groups
-        })
+        "regex": Box(match.groupdict())
```

Comment:
I reckon they might be okay without being keyed on the dict, i.e. they could just be top level variables? so you can just return the `Box(match.groupdict())`

Reply:
This follows the pattern of the other helpers. Also returning the matches as top level variables increases chance some other existing variable being overwritten inadvertently.